app/helpers/injection_helper.rb

```diff
@@ -51,6 +51,18 @@ def inject_enterprise_attributes
     render partial: "json/injection_ams", locals: {name: 'enterpriseAttributes', json: "#{@enterprise_attributes.to_json}"}
   end
 
+  def inject_orders_by_distributor
+    # Convert ActiveRecord::Relation to array for serialization
+    data_array = spree_current_user.orders_by_distributor.to_a
+    data_array.each do |enterprise|
```

Comment:
I was wondering if we could do this filtering on database level in `Spree::User.orders_by_distributor`. That would require a left join, I think.